### ISSUE TITLE
No log in button

### ISSUE THREAD

**Issue by WarmongeringBeaver** [bitwarden/clients#9261]
### Steps To Reproduce

When opening my desktop app I simply do not see any "log in" button:

Restarting Bitwarden does not fix the issue. Resizing the window makes no effect.

I can log out and then log in from scratch again thought.

### Expected Result

A login button.

### Actual Result

No log in button.

### Screenshots or Videos

![Capture d’écran du 2024-05-20 11-19-24](https://github.com/bitwarden/clients/assets/46932488/5479108d-7e4c-4a27-bb50-2078d1571ef3)

### Additional Context

Was working some time ago but can't remember when.

### Operating System

Linux

### Operating System Version

Manjaro 24.0.0, Linux 6.9.0-1-MANJARO, X11, GNOME 46.1

### Installation method

Other

### Build Version

Version 2024.4.3, Shell 28.3.1, Renderer 120.0.6099.291, Node 18.18.2, Architecture x64

### Issue Tracking Info

- [X] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by daniellbw** [comment]
Hi @WarmongeringBeaver 

Could you please determine the following. How did you install Bitwarden Desktop Client

Could you click on View -> Reload and see if that helps the matter?

If the interface you have provided us with a screenshot was working as expected - would you be prompted for a master password or a PIN in this case?

**Comment by WarmongeringBeaver** [comment]
Hello @daniellbw,

Bitwarden was installed from Manjaro's repositories, most probably with the pamac manager.

Reloading the view does not change anything.

I should be prompted for a master password.

**Comment by daniellbw** [comment]
Hi @WarmongeringBeaver 

Could you confirm your installation method for me?

Could you try uninstalling the desktop client and using a different installation method(mentioning what that is) and confirm if the issue still persists?

**Comment by daniellbw** [comment]
Hi @WarmongeringBeaver 

I will be closing this as a duplicate of the issue #8732 

Please could you provide as much information as you can regarding the method you used to install Bitwarden as well as whether you only use a master password to unlock your vault or whether you use a PIN or Biometrics as this would help us investigate this further

Looking forward to hearing from you!

**Comment by WarmongeringBeaver** [comment]
Hello, 

I confirm that Bitwarden was installed with pamac, and I use a master password at startup then a PIN code at vault lock after delay (no biometrics).

The issue seems gone for now and I can't reproduce it, though it did happen more than once in the past for sure and also disappeared randomly (but only after reboot I believe).

#8732 looks very similar so I think it's ok to close this one indeed.

Good luck with that nasty graphical bug :)
